docs/log_metric_callback.rst

```diff
@@ -30,9 +38,53 @@ Define the function with the follow signature:
         """
         your code 
 
+
+Example of a Metric Callback
+============================
+
+This example shows how to use MLFlow logger to log metrics:
+
+.. code-block:: python
+
+    import mlflow
+
+    def callback_name(node_name, task_name, metric_name, metric, round_number):
+        """
+        Write metric callback 
+
+        Args:
+            node_name (str): Name of node, which generate metric 
+            task_name (str): Name of task
+            metric_name (str): Name of metric 
+            metric (np.ndarray): Metric value
+            round_number (int): Round number
+        """
+        mlflow.log_metrics({f'{node_name}/{task_name}/{metric_name}': float(metric), 'round_number': round_number})
+
+You could view the log results either through UI interactively by typing :code:`mlflow ui` or through the use of :code:`MLflowClient`. By default, only the last logged value of the metric is returned. 
+If you want to retrieve all the values of a given metric, uses :code:`mlflow.get_metric_history` method.
+
+.. code-block:: python
+
+    import mlflow
+    client = mlflow.tracking.MlflowClient()
+    print(client.get_metric_history("<RUN ID>", "validate_local/locally_tuned_model_validation/accuracy"))
+
+  
 Command Line Interface
 ======================
 
+For logging through Tensorboard, enable the parameter :code:`write_logs : true` in `aggregator's plan settings <https://github.com/intel/openfl/blob/develop/openfl-workspace/workspace/plan/defaults/aggregator.yaml>`_ :
+
+.. code-block:: yaml
+
+  aggregator :
+    template : openfl.component.Aggregator
+    settings :
+        write_logs : true
+
+You can also write your custom callback function instead:
+
 1. Define the callback function, like how you defined in Python API, in the **src** directory in your workspace.
 
 2. Provide a way to your function with the ``log_metric_callback`` key in the ``aggregator`` section of the **plan.yaml** file in your workspace. 
```

Comment:
If we have a template that implements this callback it should be mentioned here

Reply:
Moved that description here from the end of the file.